Improvements to build process for OSX build support

thornley-touchstar opened the pull request:
Requires updated bash and gnu coreutils, gnu-sed via brew.
Added HostOS platform sanity check (only linux and osx supported/tested)
Makefile and u-boot/Makefile forces SHELL to bash which addresses issues with cross platform "echo" and nullifies previous workaround /bin/echo (for -e and -n).
Added documentation for building on OS X

pepe2k commented:
Merged with minor changes, thanks for contribution and sorry for the delay!

Cheers,
Piotr

thornley-touchstar commented:
No problems, as I mentioned I have one more pull request to send you.

I cleaned up some compilation warnings and added improved support for lzma on linux and osx.

Will send the pull request to you later today ☺

Thanks Piotr!

Regards,

David.

From: Piotr Dymacz [mailto:notifications@github.com]
Sent: Wednesday, June 15, 2016 7:33 PM
To: pepe2k/u-boot_mod <u-boot_mod@noreply.github.com>
Cc: David Thornley <David.Thornley@touchstargroup.com>; Author <author@noreply.github.com>
Subject: Re: [pepe2k/u-boot_mod] Improvements to build process for OSX build support (#104)


Merged with minor changes, thanks for contribution and sorry for the delay!

Cheers,
Piotr

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/pepe2k/u-boot_mod/pull/104#issuecomment-226136677>, or mute the thread<https://github.com/notifications/unsubscribe/AH7pzP2IL3aUQKx46164HBehRX_DCg_Sks5qL8bkgaJpZM4IqGyf>.
